ui/lib/kv/addon/routes/secret/metadata.js

```diff
@@ -20,44 +20,22 @@ export default class KvSecretMetadataRoute extends Route {
     });
   }
 
-  async fetchCapabilities(backend, path) {
```

Comment:
made more sene to only make on request to sys/capabilities from the `secret.js` route